src/test/clojure/clara/test_rules.clj

```diff
@@ -3004,3 +3004,23 @@
 
       (is (= {:?w 10, :?t nil}
              (-> e (ex-data) :bindings))))))
+
+;; Validate that the inital vector will remain as a vector even after a retraction occurs
+;; See https://github.com/cerner/clara-rules/issues/338
```

Comment:
Should we do similar validation that if a user adds an empty list as the initial value (instead of the default empty vector) that we'll keep that type as well?  I'm inclined toward yes, but don't feel strongly about it right now.  By "add" I mean something like:

````
(assoc (acc/all) :initial-value ())
````